ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/basic/TestXAROverride.java

```diff
@@ -0,0 +1,136 @@
+package com.hp.mwtests.ts.jta.basic;
+
+import com.arjuna.ats.internal.jta.utils.XAUtils;
+import com.arjuna.ats.jta.common.jtaPropertyManager;
+import org.junit.Test;
+
+import javax.transaction.NotSupportedException;
+import javax.transaction.RollbackException;
+import javax.transaction.SystemException;
+import javax.transaction.xa.XAException;
+import javax.transaction.xa.XAResource;
+import javax.transaction.xa.Xid;
+import java.util.Collections;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TestXAROverride {
```

Comment:
Maybe a sentence of a comment what is this test testing could be benefical, but not necessary. The name of the test says a bit.